### ISSUE TITLE
Cypher Query processing crashes the server

### ISSUE THREAD

**Issue by mdrobulis** [neo4j/neo4j#11408]
## Cypher Query processing error

```
Neo.DatabaseError.General.UnknownError 
For input string: "7bfd66b4"
```

while executing this query
```
with [e85814d5-49a8-40c5-803b-14996501b662, e85814d5-49a8-40c5-803b-14996501ce11, e85814d5-49a8-40c5-803b-14996501c543, e93799e2-6e7a-42d1-a2bd-4b0ec1c70aed, b868c4b6-3443-4c68-90f4-d387b5993595, e93799e2-6e7a-42d1-a2bd-4b0ec1c70b84, e93799e2-6e7a-42d1-a2bd-4b0ec1c70b2a, e93799e2-6e7a-42d1-a2bd-4b0ec1c703ab, e93799e2-6e7a-42d1-a2bd-4b0ec1c70331, b0c386e4-5439-4021-87ba-8f74236642cd, e93799e2-6e7a-42d1-a2bd-4b0ec1c70005, b868c4b6-3443-4c68-90f4-d387b5993375, b868c4b6-3443-4c68-90f4-d387b5993320, b868c4b6-3443-4c68-90f4-d387b59933cc, b868c4b6-3443-4c68-90f4-d387b5993413, b868c4b6-3443-4c68-90f4-d387b59934d5, b868c4b6-3443-4c68-90f4-d387b5993555, b868c4b6-3443-4c68-90f4-d387b599352d, b868c4b6-3443-4c68-90f4-d387b59935ed, b868c4b6-3443-4c68-90f4-d387b5992f23, b868c4b6-3443-4c68-90f4-d387b5992fb1, b868c4b6-3443-4c68-90f4-d387b599307b, b868c4b6-3443-4c68-90f4-d387b5993171, b868c4b6-3443-4c68-90f4-d387b5993117, b868c4b6-3443-4c68-90f4-d387b599318e, b868c4b6-3443-4c68-90f4-d387b5993264, b868c4b6-3443-4c68-90f4-d387b59932b0, e93799e2-6e7a-42d1-a2bd-4b0ec1c7083c, b868c4b6-3443-4c68-90f4-d387b5992eb5, b868c4b6-3443-4c68-90f4-d387b5992ee6, b868c4b6-3443-4c68-90f4-d387b5992d86, b868c4b6-3443-4c68-90f4-d387b5992db7, b868c4b6-3443-4c68-90f4-d387b5992d05, 1641cbfa-45e6-426b-a799-26485c09633c, 1641cbfa-45e6-426b-a799-26485c096390, 1641cbfa-45e6-426b-a799-26485c096353, 1641cbfa-45e6-426b-a799-26485c096369, 7bfd66b4-4659-43fa-886e-e5eacfa3fff8, 38a84960-b265-45c1-b133-8fe3a3d81c77, 7bfd66b4-4659-43fa-886e-e5eacfa3f77f, 7bfd66b4-4659-43fa-886e-e5eacfa3d2c0, 54f66f1b-8d52-4e3c-abea-7b79f1bd22ff, 7bfd66b4-4659-43fa-886e-e5eacfa3f479, ce7203a5-e99f-4c2e-ad4f-38da8d06ac79, af6d9161-c889-4da4-98fc-22f78c6e3e59, f349c2c7-15bb-477a-b5ee-878b1fe0d36f, f349c2c7-15bb-477a-b5ee-878b1fe0d343, e85814d5-49a8-40c5-803b-14996501b505, 6b7b7d58-6fdd-46b0-93ea-2f59a7930c04, af6d9161-c889-4da4-98fc-22f78c6e20ad, 09a636df-3c24-499a-b5ae-95108dc6dc8d, 54f66f1b-8d52-4e3c-abea-7b79f1bd1c49, af6d9161-c889-4da4-98fc-22f78c6e200c, 54f66f1b-8d52-4e3c-abea-7b79f1bd1d8a, 54f66f1b-8d52-4e3c-abea-7b79f1bd1d9a, 54f66f1b-8d52-4e3c-abea-7b79f1bd1ab4, 407ae21e-287f-4813-8320-0fcf1d08e8fc, 3a0507a5-5221-44e8-b693-354b032b229d, 0f120df8-58c0-4aad-887b-547d69caf7b1, 20629b2a-6a93-4fd4-864b-32c3d6a996a1, 20629b2a-6a93-4fd4-864b-32c3d6a99d88, 20629b2a-6a93-4fd4-864b-32c3d6a9913d, 889d86bf-6b48-4a61-b7e0-3ec63dc45b2e, 889d86bf-6b48-4a61-b7e0-3ec63dc45b2a, 20629b2a-6a93-4fd4-864b-32c3d6a8620c, af6d9161-c889-4da4-98fc-22f78c6ede61, 6380643d-d929-4ac9-ad3f-16000d7c7984, e594bd09-ac39-4e9f-a308-3de34dc64702, e594bd09-ac39-4e9f-a308-3de34dc6477c, 636b05be-6379-43cb-868d-73f0ebdd3929, 6380643d-d929-4ac9-ad3f-16000d7c6f87, a2eb5a4b-3508-46a1-a5bf-150314853112, a2eb5a4b-3508-46a1-a5bf-15031485306f, 77145dfc-3e88-439a-9ddf-409e21a5eb0c, a2eb5a4b-3508-46a1-a5bf-150314853429, a2eb5a4b-3508-46a1-a5bf-15031485335f, a2eb5a4b-3508-46a1-a5bf-1503148530ea] as list return list
```

**Neo4j Version:** Neo4j 3.1.4
**Operating System:** Linux development 3.10.0-514.el7.x86_64 #1 SMP Tue Nov 22 16:42:41 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

### Steps to reproduce
1. Start neo4j SERVER
2. Open the web brower interface 
3. Execute command listed above   

### Expected behavior
` Neo.ClientError.Statement.SyntaxError `

Server says alive, keeps processing queries

### Actual behavior
Web UI notifies about the Unknown Error
Server HANGS

```
Stopping Neo4j.................................................................................................................                                                         .... failed to stop
Neo4j (pid 24712) took more than 120 seconds to stop.
Please see <...>/databases/staging/logs/neo4j.log for details.
```

neo4j.log content
```
2018-03-27 13:09:47.918+0000 ERROR Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4". See debug.log for more details, reference 16d8be2e-7658-43f6-a2eb-daac7a5f8967.
2018-03-27 13:09:47.945+0000 ERROR Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4". See debug.log for more details, reference e4a33149-65a4-45e1-b9c0-5bed630a7c62.
2018-03-27 13:10:04.494+0000 ERROR Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4". See debug.log for more details, reference d69ce8d4-4591-4cf6-97b4-7b30bc0ee0fe.
2018-03-27 13:29:24.907+0000 ERROR Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4". See debug.log for more details, reference 03b78563-595b-452f-b647-a6727f822030.
2018-03-27 13:29:29.865+0000 INFO  Neo4j Server shutdown initiated by request
2018-03-27 13:29:32.356+0000 INFO  Stopping...
2018-03-27 13:29:32.397+0000 ERROR Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4". See debug.log for more details, reference a2d5fbab-2ee8-4a5e-a046-b0c2368b53cb.
2018-03-27 13:29:32.417+0000 ERROR Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4". See debug.log for more details, reference b4c7d781-913a-4694-846d-70d9eb38ea22.
2018-03-27 13:29:36.024+0000 INFO  Stopped.

```

debug.log content
```
2018-03-27 13:29:32.417+0000 ERROR [o.n.b.v.r.ErrorReporter] Client triggered an unexpected error [UnknownError]: For input string: "7bfd66b4", reference b4c7d781-913a-4694-846d-70d9eb38ea22. For input string: "7bfd66b4"
java.lang.NumberFormatException: For input string: "7bfd66b4"
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
        at java.lang.Long.parseLong(Long.java:589)
        at java.lang.Long.parseLong(Long.java:631)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.DecimalIntegerLiteral.value$lzycompute(Literal.scala:45)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.DecimalIntegerLiteral.value(Literal.scala:45)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.Subtract.checkBoundary(ArithmeticFunctions.scala:147)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.Subtract.semanticCheck(ArithmeticFunctions.scala:143)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.Expression$SemanticCheckableExpressionTraversable$$anonfun$semanticCheck$3.apply(Expression.scala:49)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.Expression$SemanticCheckableExpressionTraversable$$anonfun$semanticCheck$3.apply(Expression.scala:49)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:53)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:52)
        at scala.collection.LinearSeqOptimized$class.foldLeft(LinearSeqOptimized.scala:124)
        at scala.collection.immutable.List.foldLeft(List.scala:84)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:53)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:52)
        at scala.collection.LinearSeqOptimized$class.foldLeft(LinearSeqOptimized.scala:124)
        at scala.collection.immutable.List.foldLeft(List.scala:84)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:53)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:52)
        at scala.collection.LinearSeqOptimized$class.foldLeft(LinearSeqOptimized.scala:124)
        at scala.collection.immutable.List.foldLeft(List.scala:84)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:53)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2$$anonfun$apply$1.apply(SemanticCheck.scala:52)
        at scala.collection.TraversableOnce$$anonfun$foldLeft$1.apply(TraversableOnce.scala:157)
        at scala.collection.TraversableOnce$$anonfun$foldLeft$1.apply(TraversableOnce.scala:157)
        at scala.collection.Iterator$class.foreach(Iterator.scala:893)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
 at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
        at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
        at scala.collection.TraversableOnce$class.foldLeft(TraversableOnce.scala:157)
        at scala.collection.AbstractTraversable.foldLeft(Traversable.scala:104)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.TraversableOnceSemanticChecking$$anonfun$foldSemanticCheck$extension$2.apply(SemanticCheck.scala:51)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:61)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.SingleQuery$$anonfun$checkClauses$1$$anonfun$4.apply(Query.scala:119)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.SingleQuery$$anonfun$checkClauses$1$$anonfun$4.apply(Query.scala:117)
        at scala.collection.LinearSeqOptimized$class.foldLeft(LinearSeqOptimized.scala:124)
        at scala.collection.immutable.List.foldLeft(List.scala:84)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.SingleQuery$$anonfun$checkClauses$1.apply(Query.scala:117)
        at org.neo4j.cypher.internal.frontend.v3_1.ast.SingleQuery$$anonfun$checkClauses$1.apply(Query.scala:116)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:61)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:60)
        at org.neo4j.cypher.internal.frontend.v3_1.ChainableSemanticCheck$$anonfun$chain$extension$1.apply(SemanticCheck.scala:59)
        at org.neo4j.cypher.internal.compiler.v3_1.SemanticChecker.check(SemanticChecker.scala:29)
        at org.neo4j.cypher.internal.compiler.v3_1.CypherCompiler$$anonfun$8.apply(CypherCompiler.scala:201)
        at org.neo4j.cypher.internal.compiler.v3_1.CypherCompiler$$anonfun$8.apply(CypherCompiler.scala:201)
        at org.neo4j.cypher.internal.compiler.v3_1.helpers.package$$anonfun$closing$1.apply(package.scala:29)
        at org.neo4j.cypher.internal.compiler.v3_1.helpers.package$$anonfun$closing$1.apply(package.scala:29)
        at org.neo4j.cypher.internal.compiler.v3_1.helpers.package$.using(package.scala:37)
        at org.neo4j.cypher.internal.compiler.v3_1.helpers.package$.closing(package.scala:29)
        at org.neo4j.cypher.internal.compiler.v3_1.CypherCompiler.prepareSyntacticQuery(CypherCompiler.scala:200)
        at org.neo4j.cypher.internal.compatibility.CompatibilityFor3_1$$anonfun$3.apply(CompatibilityFor3_1.scala:237)
        at org.neo4j.cypher.internal.compatibility.CompatibilityFor3_1$$anonfun$3.apply(CompatibilityFor3_1.scala:237)
        at scala.util.Try$.apply(Try.scala:192)
        at org.neo4j.cypher.internal.compatibility.CompatibilityFor3_1$class.produceParsedQuery(CompatibilityFor3_1.scala:237)
        at org.neo4j.cypher.internal.compatibility.CompatibilityFor3_1Cost.produceParsedQuery(CompatibilityFor3_1.scala:492)
        at org.neo4j.cypher.internal.CypherCompiler.parseQuery(CypherCompiler.scala:141)
 at org.neo4j.cypher.internal.ExecutionEngine$$anonfun$parsePreParsedQuery$1.apply(ExecutionEngine.scala:117)
        at org.neo4j.cypher.internal.ExecutionEngine$$anonfun$parsePreParsedQuery$1.apply(ExecutionEngine.scala:116)
        at scala.Option.getOrElse(Option.scala:121)
        at org.neo4j.cypher.internal.ExecutionEngine.parsePreParsedQuery(ExecutionEngine.scala:116)
        at org.neo4j.cypher.internal.ExecutionEngine.org$neo4j$cypher$internal$ExecutionEngine$$producePlan$1(ExecutionEngine.scala:166)
        at org.neo4j.cypher.internal.ExecutionEngine$$anonfun$liftedTree1$1$1.apply(ExecutionEngine.scala:170)
        at org.neo4j.cypher.internal.ExecutionEngine$$anonfun$liftedTree1$1$1.apply(ExecutionEngine.scala:170)
        at org.neo4j.cypher.internal.compiler.v3_1.QueryCache$$anonfun$getOrElseUpdate$1$$anonfun$apply$1.apply(CacheAccessor.scala:36)
        at org.neo4j.cypher.internal.compiler.v3_1.MonitoringCacheAccessor$$anonfun$1.apply(CacheAccessor.scala:57)
        at org.neo4j.cypher.internal.compiler.v3_1.LFUCache$$anon$1.apply(LFUCache.scala:31)
        at com.github.benmanes.caffeine.cache.BoundedLocalCache.lambda$doComputeIfAbsent$16(BoundedLocalCache.java:1973)
        at java.util.concurrent.ConcurrentHashMap.compute(ConcurrentHashMap.java:1892)
        at com.github.benmanes.caffeine.cache.BoundedLocalCache.doComputeIfAbsent(BoundedLocalCache.java:1971)
        at com.github.benmanes.caffeine.cache.BoundedLocalCache.computeIfAbsent(BoundedLocalCache.java:1954)
        at com.github.benmanes.caffeine.cache.LocalCache.computeIfAbsent(LocalCache.java:113)
        at com.github.benmanes.caffeine.cache.LocalManualCache.get(LocalManualCache.java:54)
        at org.neo4j.cypher.internal.compiler.v3_1.LFUCache.getOrElseUpdate(LFUCache.scala:30)
        at org.neo4j.cypher.internal.compiler.v3_1.LFUCache.apply(LFUCache.scala:48)
        at org.neo4j.cypher.internal.compiler.v3_1.MonitoringCacheAccessor.getOrElseUpdate(CacheAccessor.scala:55)
        at org.neo4j.cypher.internal.compiler.v3_1.QueryCache$$anonfun$getOrElseUpdate$1.apply(CacheAccessor.scala:34)
        at scala.collection.Iterator$$anon$9.next(Iterator.scala:162)
        at scala.collection.Iterator$$anon$12.nextCur(Iterator.scala:434)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:440)
        at scala.collection.Iterator$$anon$12.next(Iterator.scala:444)
        at org.neo4j.cypher.internal.compiler.v3_1.QueryCache.getOrElseUpdate(CacheAccessor.scala:46)
        at org.neo4j.cypher.internal.ExecutionEngine.liftedTree1$1(ExecutionEngine.scala:170)
        at org.neo4j.cypher.internal.ExecutionEngine.planQuery(ExecutionEngine.scala:156)
        at org.neo4j.cypher.internal.ExecutionEngine.execute(ExecutionEngine.scala:106)
        at org.neo4j.cypher.internal.javacompat.ExecutionEngine.executeQuery(ExecutionEngine.java:59)
        at org.neo4j.bolt.v1.runtime.TransactionStateMachineSPI$1.start(TransactionStateMachineSPI.java:135)
        at org.neo4j.bolt.v1.runtime.TransactionStateMachine$State$1.run(TransactionStateMachine.java:187)
        at org.neo4j.bolt.v1.runtime.TransactionStateMachine.run(TransactionStateMachine.java:77)
        at org.neo4j.bolt.v1.runtime.BoltStateMachine$State$2.run(BoltStateMachine.java:396)
        at org.neo4j.bolt.v1.runtime.BoltStateMachine.run(BoltStateMachine.java:196)
        at org.neo4j.bolt.v1.messaging.BoltMessageRouter.lambda$onRun$3(BoltMessageRouter.java:80)
        at org.neo4j.bolt.v1.runtime.concurrent.RunnableBoltWorker.execute(RunnableBoltWorker.java:135)
        at org.neo4j.bolt.v1.runtime.concurrent.RunnableBoltWorker.run(RunnableBoltWorker.java:89)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by Lojjs** [comment]
@mdrobulis Thanks for reporting, we in the Neo4j Cypher team will have a look.
Best regards Louise

**Comment by sherfert** [comment]
@mdrobulis The error on our end was that we swallow the actual error message and give a not very helpful `Neo.DatabaseError.General.UnknownError` instead. This is fixed in the referencing PR and will be included in the next release of 3.1

As you probably know, the problem is that the Strings are not quoted, i.e. one should write `"e85814d5-49a8-40c5-803b-14996501b662"` instead of `e85814d5-49a8-40c5-803b-14996501b662`. They got interpreted as invalid number literals and while trying to communicate that fact, it failed with another error because your dashes between numbers are interpreted as subtraction.

I could not reproduce the hanging behavior. It would be nice if you could re-evaluate once the fix is in and open another issue if you can still make Neo4j hang with your query.
